spark-bigquery-pushdown/pushdown_common_src/main/scala/com/google/cloud/spark/bigquery/pushdowns/SparkBigQueryPushdownUtil.scala

```diff
@@ -126,4 +130,12 @@ object SparkBigQueryPushdownUtil {
       }
     }
   }
+
+  def getTableNameFromOptions(options: Map[String, String]): String = {
```

Comment:
Please check SparkBigQueryUtil.parseSimpleTaleId().

Also, as this method is not pushdown specific, it should be in SparkBigQueryUtil